### PR TITLE
Added a check for NodeJS version

### PR DESCRIPTION
**Pull request by brian-learningpool**
This will generate a warning if not v0.10.33.

### CODE REVIEW

**Comment by brian-learningpool** [comment]
Resolves #660.

**Comment by taylortom** [comment]
+1
Did we want to have the version required pulled in from a config or anything? (not sure if this is needed anywhere else)

**Comment by brian-learningpool** [comment]
I don't think it's needed anywhere else, and could possibly be removed when we re-work the code for v0.12.

File: lib/application.js [hunk 1]
```diff
@@ -308,6 +308,11 @@ Origin.prototype.startServer = function () {
     serverOptions.minimal = true;
   }
   
+  // Check the NodeJS version 
+  if (process.version !== 'v0.10.33') {
+    logger.log('warn', 'Incorrect version of NodeJS required -- this code requires v0.10.33 to run correctly.');
```
Comment (dennis-learningpool) [inline lib/application.js]: I think the message could worded a little better, here. I suggest:
"You are running an unsupported version of NodeJS (" + process.version + "). This application has only been fully tested on NodeJS v0.10.33. It is recommended that you use that version instead."

**Comment by finbartracey1** [comment]
+1

File: lib/application.js (hunk 1 above)
Comment (brian-learningpool) [inline lib/application.js]: Me fail English?  That's unpossible.  

PR updated. ;)

**Comment by dennis-learningpool** [comment]
+1 ralphie

**Pull request merged**